bus_messages/migrations/0001_initial.py

```diff
@@ -79,8 +79,8 @@ class Migration(migrations.Migration):
                 ('alarm_and_status', models.IntegerField()),
                 ('bms_assembly_revision', models.IntegerField()),
                 ('bms_serial_no', models.CharField(max_length=16)),
-                ('bms_master_software_version', models.CharField(max_length=4)),
-                ('bms_slave_software_version', models.CharField(max_length=4)),
+                ('bms_main_software_version', models.CharField(max_length=4)),
+                ('bms_subordinate_software_version', models.CharField(max_length=4)),
```

Comment:
This is great for ensuring new applications have these names, however it wouldn't work for existing applications.
We really need a new migration to update existing datastores that checks to see if these names have already been corrected or not.